packages/core/test/utils/safe.test.ts

```diff
@@ -38,14 +38,30 @@ describe('safe', () => {
     test('calls given function with correct args', () => {
       const mockFn = jest.fn();
       const actualSafeFunction = safeTracesSampler(mockFn);
-      actualSafeFunction?.({ name: 'foo', transactionContext: { name: 'foo' } });
+      actualSafeFunction?.({
+        name: 'foo',
+        transactionContext: { name: 'foo' },
+        inheritOrSampleWith: function (fallbackSampleRate: number): number {
+          return fallbackSampleRate;
+        },
+      });
       expect(mockFn).toBeCalledTimes(1);
-      expect(mockFn).toBeCalledWith({ name: 'foo', transactionContext: { name: 'foo' } });
+      expect(mockFn).toBeCalledWith({
+        name: 'foo',
+        transactionContext: { name: 'foo' },
+        inheritOrSampleWith: expect.any(Function),
```

Comment:
Let's check that it's the same function passed to the original. That will be more robust than any function.